src/Home.js

```diff
@@ -114,10 +114,13 @@ const responsiveStyles = [
 
 const commonStyles = StyleSheet.create({
     container: {
+        display:'flex',
+        minHeight: '100%',
```

Comment:
Not sure if this is still required.

Reply:
yes they are not needed now.

Reply:
Please remove it and confirm whether the website still works.